app/controllers/account/tasks_controller.rb

```diff
@@ -121,7 +121,8 @@ def uncomplete
   private
 
   def parent
-    current_user.projects.find(params[:project_id])
+    return Project.find(params[:project_id]) if current_user.available_workspaces.
```

Comment:
basically you need something like:

current_user.available_projects.find(params[:project_id])